docs/concepts/schedules.md

```diff
@@ -76,8 +76,13 @@ deployments:
   schedule:
     cron: 0 0 * * *
     timezone: America/Chicago
+    active: true
```

Comment:
would it make more sense to show `active: false` since the default is True?

Reply:
Sure, that makes sense.